README.md

```diff
@@ -55,179 +45,152 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
+- [❓ FAQ ](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Summer Camp Event Page <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+> This Project is about the event going to be held in the march THE Summer Camp organized by SIA
 
-**[your_project__name]** is a...
+**Summer Camp Event Page** is just a introduction about the upcoming event 
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+> In this project I used many skills, tech and libraries
 
 <details>
-  <summary>Client</summary>
+  <summary>Documentation</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://html.com">HTML</a></li>
   </ul>
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>Styling</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://www.w3.org">CSS</a></li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+<summary>Dynamics And Logics</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://michalsnik.github.io/aos/">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Animation</summary>
+  <ul>
+    <li><a href="https://michalsnik.github.io/aos/">Aos.js</a></li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+> Following features you should observe
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Animation on scroll**
+- **Navigation list hover**
+- **Pre Navigation bar is about me**
+- **Pre Navigation bar have Google Translator for our wbsite**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+> This link will guide you to my project
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://powerlevel9000.github.io/Dynamic-Event-Template/)
+- [Loom Video](https://www.loom.com/share/263ca564cb634bcca4b48ad1ea9870b7)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Its great that you have recorded a loom video :blush: but the problem is that your video length should not be more than 5 mins but your video is more than 6 minutes :thinking:
- [ ] Your video should contain a piece of the code you find interesting or are proud of making :+1:

Reply:
# Genral Asking
- Please Can you tell me why my video should not be more than 5mins 
-  Also, I didn't make anything to be proud of yet except the website language converter

Reply:
# linters are there 

- I didn't make any changes in linters  kindly check my .github/workflows can check it
- the thing it is not showing here I don't know why maybe because it already passed or network error or git hub issue
- @PowerLevel9000
[Linter Errors](https://github.com/PowerLevel9000/Dynamic-Event-Template/pull/1/commits/784c3324e2e8194a36cb6571367d27f4f07269ec)

kindly check this commit

and also next to it  I didn't make any changes in linters

Reply:
ok For now I am making invalid changes and pushing it back Let's see what happens

Reply:
Hey @PowerLevel9000 I hope you are doing well :blush:

**Que: Why the video length should be limited to 5 mins?** 

Ans: Its because it is a requirement in your project. Whenever you will have to make a presentation video then you will be given a limited time to present your project so that's why Microverse gave you 5 mins to present your project.
Whatever you make you should show that code in your video :+1:

Reply:
Hi @HAMMAS-SALEEM,

I hope you're doing well. I wanted to address something that might come across as offensive or argumentative in my previous message. I apologize if my tone wasn't good, especially considering that I was a junior and still working on developing my professional skills. I want to assure you that I didn't had any ill intentions, as feelings are difficult to convey through text. I hope you understand and accept my apology.

Similarly, I would like to extend the same sentiment to @anagudelogu. I hope you understand that my intention was not to offend or argue, but rather to express my thoughts.

Thank you for your understanding.

Happy Coding 

@PowerLevel9000